package/MDAnalysis/coordinates/PDB.py

```diff
@@ -493,6 +500,11 @@ class PDBWriter(base.WriterBase):
     are not set (:code:`None` or :code:`np.zeros(6)`,
     see Issue #2698).
 
+    When the record_types attribute is present (e.g. Universe object was
```

Comment:
Same as other comment, we probably should use:
```
:attr:`record_types`
```

Reply:
Done. Also current develop merged (CHANGELOG conflict).